content/blog/return0e/2019-09-19_nodejs_now_available_in_haiku.md

```diff
@@ -0,0 +1,57 @@
+
+To install Node.js on Haiku, search for the package in HaikuDepot and click 'Install' or type ``pkgman install nodejs`` into the Haiku Terminal and it will automatically install everything.
+
+So what does this actually mean for us users? Well you can now directly install and run software from the JavaScript and TypeScript ecosystems using npm (Node Package Manager) or yarn and most packages will work right out of the box. Here are some example packages that are running on the system:
+
+React and TypeScript:
+
+<img src="https://user-images.githubusercontent.com/7050293/65165254-54f5e100-da36-11e9-8b2f-8710ca6ff46e.png" alt="react-and-typescript" class="img-responsive center-block">
+
+
+Angular and TypeScript:
+
+<img src="https://user-images.githubusercontent.com/7050293/65165285-62ab6680-da36-11e9-8f9c-9f7d33ff25de.png" alt="angular-demo" class="img-responsive center-block">
+
+
+Simple Express Web Server:
+
+<img src="https://user-images.githubusercontent.com/7050293/65165316-72c34600-da36-11e9-88fb-e2ddb120bc9c.png" alt="expressjs" class="img-responsive center-block">
+
+
+Socket.IO Chat Demonstration:
+
+<img src="https://user-images.githubusercontent.com/7050293/65165337-7e167180-da36-11e9-8ab8-71505a69e0ba.png" alt="socketio" class="img-responsive center-block">
+
+
+Creating a static site via Hexo:
+
+<img src="https://user-images.githubusercontent.com/7050293/65165369-8c648d80-da36-11e9-9385-079aeb19a5a4.png" alt="hexo" class="img-responsive center-block">
+
+It is also possible to attach node programs for [remote debugging](https://imgur.com/a/FEFOAvu) via the Chrome Inspector remotely.
+
+Please note that since this is a very early port to a new operating system, it is expected that several packages require platform recognition for it to function correctly. Therefore, some packages will not work unmodified and would require patches. In relation to this, some features in this port are currently unavailable such as file system events _(fs-events)_ which is used for watching file changes in the file-system is unimplemented. While Haiku itself has the ability to monitor file system events, there is a plan to integrate this into the Node port in the future.
```

Comment:
Remove "is unimplemented" from second to last sentence.

Reply:
@humdingerb done.